platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/LogoutState.java

```diff
@@ -33,8 +35,13 @@
 
   private boolean partialLogout = false;
 
-  public LogoutState(Set<String> spDescriptors) {
+  private List<String> sessionIndexes = Collections.emptyList();
```

Comment:
:question: Is there any concern with storing state like this in an HA or HS environment?

Reply:
Great question @figliold. Don't think we run the IdP in HA configuration though, do we?

Reply:
Do we have an alternative place to store it if we do need to?

Reply:
In a HA configuration you shouldn't need more than one IDP. If you do have multiple IDPs, the SAML Response point to the IDP that issued the Assertion in `<Issuer></Issuer>`